TUTORIAL-ASSETS.md

```diff
@@ -110,22 +109,21 @@ mapX,mapY,pixelX,pixelY,enable
 63084.8731,27443.9360,-2.9792483,56.4622088,1
 ```
 
-This describes four points - the first two coordinates of each line specify X and Y coordinates in the DXF file, while the next two coordinates are a longitude and latitude.  Looking at our example DXF file and floorplan GeoJSON, the 4 points refer to the following:
+This describes four points - the first two coordinates of each line specify X and Y coordinates in the DXF file, while the next two coordinates are a longitude and latitude.  Looking at our example DXF file and floor plan GeoJSON, the 4 points refer to the following:
 
 ![Image showing the positions of the 4 georeference coordinates](images/cadtutorial/georeferencing.png)
 
-> ## NOTE 
->> You can add additional points as necessary - generally the more points you map, the more accurate the extraction process will be.  For a small,  rectangular floorplan like this, 4 are enough!
+> :information_source: You can add additional points as necessary - generally the more points you map, the more accurate the extraction process will be. For a 
 
 ### Creating the config.json file
 
 In our example, we'll be extracting the highlighted desks and representing them with a similar 3D model:
 
 ![Image showing the desks we'll be extracting from the CAD](images/cadtutorial/desks_cad.png)
 
-The config.json file part of our submission details how the CAD will be read by the extraction process - essentially telling it which parts are desks, chairs etc. along with how to determine their orientation and what 3D Model to use to represent the Asset in your map.  Our example config file looks like this:
+The config.json file part of our submission details how the CAD will be read by the extraction process - essentially telling it which parts are desks, chairs etc. along with how to determine their orientation and what 3D Model to use to represent the Asset in your map. Our example config file looks like this:
```

Comment:
lower case "model". Generally Asset is capitalised and "3D model" is lower case which I think is fine.